src/workloads/scale/ManyUpdate.yml

```diff
@@ -0,0 +1,72 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-replication"
+Description: |
+  This workload loads a large number of small documents into a single collection, and then does a
+  small multi-update which touches all of them.  It is intended to measure the impact of the load of
+  replication of many oplog entries on the primary.  Number of documents should be significantly
+  greater than the maximum replication batch size of 5K (50K is a good minimum).
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 1000
+
+Actors:
+- Name: Setup
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &DB test
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand: {dropDatabase: 1}
+  - &Nop {Nop: true}
+  - *Nop
+
+- Name: Loader
+  Type: Loader
+  Threads: 10
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: *DB
+    CollectionCount: 1
```

Comment:
With CollectionCount of 1, only one thread will do anything. It's best to either drop the Threads to 1 (in both places) or set `multipleThreadsPerCollection` to true.

Reply:
Done

Reply:
I think you may need to remove the phase level threads statement with MultipleThreadsPerCollection: true.